app/api/wallet/network.ts

```diff
@@ -24,3 +24,25 @@ export function getBip32Option (envNetwork: EnvironmentNetwork): Bip32Options {
     wif: network.wifPrefix
   }
 }
+
+export function getTxURLByNetwork (network: EnvironmentNetwork, txId: string): string {
+  const baseUrl = new URL('https://defiscan.xyz/tx/')
```

Comment:
On second thought thinking we should structure it as such:

`https://defichain.xyz/transactions/${txid}?network=${network}`